Gemfile.lock

```diff
@@ -4,7 +4,7 @@ PATH
     heartcheck-redis (1.0.2)
       heartcheck (>= 1.0.0)
       net-telnet (~> 0.1.1)
-      redis (>= 3.2.0, < 3.4.0)
+      redis (>= 3.2.0, < 5)
```

Comment:
Given this range of accepted redis versions, do you think it is a good idea to change https://github.com/locaweb/heartcheck-redis/blob/master/.travis.yml to run the specs with redis 3.x and 4.x ?

Reply:
To do it, maybe it would be necessary to have 2 different Gemfile.lock

Reply:
We already have 2 gemfiles because of rack 2.0 not working with rubies < 2.2.something.

No problem adding another